### PR TITLE
Fixes circuitry, allowing council to re-enable printing

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

Fixes circuits, putting an end to all powergaming that may arise from it. 

#### Changelog

:cl:  
fix: Circuits are better now :)
/:cl:

### CODE REVIEW

**Review by ThatLing — APPROVED** [review]

**Comment by GuppyLaxx** [comment]
speed merge

**Comment by GuppyLaxx** [comment]
yes

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
no green dot no merge

File: code/modules/integrated_electronics/core/printer.dm [hunk 1]
```diff
@@ -34,6 +34,7 @@
 /obj/item/integrated_circuit_printer/Initialize()
 	. = ..()
 	AddComponent(/datum/component/material_container, list(MAT_METAL), MINERAL_MATERIAL_AMOUNT * 25, TRUE, list(/obj/item/stack, /obj/item/integrated_circuit, /obj/item/electronic_assembly))
+	qdel(src) //yogs - fixes circuits
```
Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [inline code/modules/integrated_electronics/core/printer.dm:37]: remove this and it's good to go

**Review by 1fbff5f83b23d39d38b1dfcb4cac8d9b — DISMISSED** [review]

**Pull request closed without merging**